src/Honeybee_Honeybee.py

```diff
@@ -1193,7 +1191,7 @@ def objToRAD(self, objFile):
         # create rad file
         
         if self.pattern != None:
-            cmd = self.RadianceFolder + "\\obj2mesh -a " + matFile + " " + objFile + " > " +  mshFile
```

Comment:
@antonszilasi you're overwriting a couple of lines which you should not. Please get this fixed and send another PR. Also you need to update the compatibleHBVersion in the components which are using this new functionalities otherwise we will have version incompatibility issues. I haven't noticed that when I was reviewing the previous PR.